File: LeetCode Solutions/December 2021 LeetCoding Challenge/Consecutive Characters/Consecutive Characters.cpp
class Solution {
public:
    int maxPower(string s) {
        int count = 1;
        int power = 1;
        for (int i=0; i<s.size() - 1; i++){
            if (s[i] == s[i+1]){
                count++;
            } else{
                count = 1;
            }
            power = max(power, count);
        }
        return power;
    }
};

File: LeetCode Solutions/November 2019 LeetCoding Challenge/Consecutive Characters.py
class Solution:
    def maxPower(self, s: str) -> int:
        count = 1
        power = 1
        for i in range(len(s) - 1):
            if(s[i] == s[i + 1]):
                count += 1
            else:
                count = 1
            power = max(power, count)
        return power